json-java8/shared/src/main/scala/rapture/json/java8/time.scala

```diff
@@ -0,0 +1,81 @@
+package rapture.json.java8
```

Comment:
Should this be in the `rapture.json` package, or would it be better as a `rapture.core.java8` package?